### DISCUSSION TITLE
hello all

### DISCUSSION THREAD

**Discussion by Ahmed12ibad** [community/community#39570] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

how are you?

**Comment by Arastoi** [comment]
Thank you friend for trusting us. for sure We will compensate you for what
you did

در تاریخ یکشنبه 20 نوامبر 2022،‏ 13:40 muhammad ahmed <
***@***.***> نوشت:

**Comment by luutrutatcavaomaytinh** [comment]
> ### Select Topic Area
> 
> General
> 
> ### Body
> 
> how are you?

**Comment by DSDmark** [comment]
Hi @Ahmed12ibad,

Change your category into **welcome center** or **new to GitHub**
   
> Nice to meet you and welcome to this great community, we are here to help each other and try to make open source better. This is the discussions section if you want to discuss any. you can ask here, we will try to help.

Or why any chance if you are curious about, 

> what is GitHub, who is GitHub then I highly suggest you check this out [**What is GitHub?**](https://youtu.be/pBy1zgt0XPc).  It's a video. so, just ***sit back and relax***

**Comment by mouradcherif** [comment]
fine but what is the reason for this discussion ? do you have a problem with your profile?
